include/tvm/runtime/memory_manager.h

```diff
@@ -37,9 +37,8 @@
 
 namespace tvm {
 namespace runtime {
-namespace vm {
 
-struct Buffer {
+struct MBuffer {
```

Comment:
It would be useful if we can move to a new namespace runtime::memory and keep the same buffer name.